src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/hearingsbyhearingtype/HearingsByHearingTypeReport.java

```diff
@@ -69,6 +71,60 @@ private void executeReport(ListingDetails listingDetails, List<SubmitEvent> subm
 
     }
 
+    private void initReport(ListingDetails listingDetails) {
+
+        var adhocReportType = new AdhocReportType();
+
+        //LocalReportsSummary fields
```

Comment:
Why put a comment when your code can document itself? Create a method called initSummaryFields. It'll make initReport easier to understand too

Reply:
refactored